kata-1/variables.tf

```diff
@@ -0,0 +1,44 @@
+variable "namespace" {
+  description = "Project namespace for unique resource naming"
+  type        = string
+}
+
+
+variable "region" {
+  description = "AWS Region"
+  default     = "us-east-1"
+  type        = string
+}
+
+variable "az" {
+  description = "AWS Availability Zone"
+  default     = "us-east-1a"
+  type        = string
+}
+
+variable "ami" {
+  description = "Amazon Machine Image"
+  type        = string
+  default     = "ami-0947d2ba12ee1ff75" # Amazon Linux 2 AMI (HVM), SSD Volume Type

```

Comment:
though hard coding works for us-east-1, it will fail for other regions. consider using a datasource instead.

Reply:
I used the AWS SSM data source to address this use case. Added on [this](https://github.com/adriaanbd/terraform_aws_dojo/pull/1/commits/6f23dbb925cbbf15bc27393e66c82f6cc57ea508) commit.